test/testcafe/spec/IdentifyWithPassword_spec.js

```diff
@@ -149,6 +149,23 @@ test.meta('v3', false).requestHooks(identifyRequestLogger, identifyWithPasswordM
   await t.expect(await identityPage.hasShowTogglePasswordIcon()).notOk();
 });
 
+test.requestHooks(identifyRequestLogger, identifyWithPasswordMock)('should auto-hide password after 30 seconds', async t => {
```

Comment:
Would appreciate some eyes on this test since it's my first that I've added with `wait()`s before. I saw other tests use a straight up `t.wait(30500)` for a 30 second timeout, but I thought adding the extra buffer time to the short `timeout` option for the `expect()` call made more sense because it overrides the longer default wait time for `expect()` calls. Otherwise, the default wait time for the assertion counts toward the total wait time and this test could pass even with something like `t.wait(20000)`

Reply:
We might want to revisit this soon and see if doing this in a unit test is the right place. Nothing urgent we need to handle but waiting 30s of wall clock time is costly. Ideally we can use the fake timers functionality in jest to ensure this works but we have to move it to a unit test that renders a view with the password field and does the same thing.

Reply:
Yeah the 30s wall clock was what I wasn't sure about. Probably should've waited to merge until I get a definitive response on this test, apologies. I can create a task ticket to revisit this test

Reply:
no worries, I was just going to add a comment saying I should have reviewed sooner. Create the ticket, add to our triage and let's revisit soon.